modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,524 @@
+= Add a BYOC Customer-Managed VPC on GCP
+:description: Use the Redpanda UI to create a customer-managed VPC peering connection for a BYOC cluster.
+:page-cloud: true
+
+You can connect a Redpanda Cloud BYOC cluster with your existing virtual private cloud (VPC). Using your own VPC gives you control over your security policies and resources, including service accounts, IAM roles, subnetworks, firewall rules, and storage buckets. 
+
+== Prerequisites
+
+* A GCP account
+* A running BYOC cluster in GCP. See xref:./create-byoc-cluster-gcp.adoc[Create a BYOC Cluster on GCP]. Using your own VPC, you select the *Customer Managed* network connection type.
+* An appropriate CIDR range: pods require /21, services require /24, GKE requires /28.
+* A project with access to the internet, and an agent with public access. The VPC and the Redpanda cluster can be in the same GCP project or in different GCP projects.
```

Comment:
The agent will not yet exist so it might be confusing to say it needs public access. It's more that they need a VPC and a Subnet and reserved CIDR ranges for pods, services, and gke... and firewall rules that grant the subnet internet access.